package.json

```diff
@@ -533,7 +533,7 @@
     "growl": "1.10.5",
     "he": "1.2.0",
     "js-yaml": "3.13.1",
-    "log-symbols": "2.2.0",
+    "log-symbols": "^3.0.0",
```

Comment:
Please remove "^", we don't use it in our dependencies.

Reply:
Oops. Didn't notice that. Fixed.